src/components/program-progress/ProgramProgressCourses.jsx

```diff
@@ -138,7 +211,16 @@ const ProgramProgressCourses = ({ courseData }) => {
       {courseData?.notStarted?.length > 0
       && (
         <div className="mb-5 courses">
-          <h4 className="white-space-pre"> REMAINING COURSES    {courseData?.notStarted?.length}</h4>
+          <h4 className="white-space-pre">
+            <FormattedMessage
+              id="enterprise.dashboard.programs.about.page.remaining.courses"
+              defaultMessage="REMAINING COURSES    {remainingCourseCount}"
```

Comment:
@muhammad-ammar Yes, the space was intentional as it was present in the original string and I retained it in the marked string.
However, I have moved the space outside the FormattedMessage since it is required.
<img width="730" alt="Screenshot 2024-02-23 at 1 17 37 PM" src="https://github.com/openedx/frontend-app-learner-portal-enterprise/assets/113524403/d0272af3-4232-457f-a412-33b96e5b6494">